### PR TITLE
Added e2e tests for `page.edited` webhook

### PR DESCRIPTION
**Pull request by shngt**
refs #15537

- Adds an e2e test and test snapshot for the `page.edited` webhook

Got some code for us? Awesome 🎊!

Please include a description of your change & check your PR against this list, thanks!

- [x] There's a clear use-case for this code change, explained below
- [x] Commit message has a short title & references relevant issues
- [x] The build will pass (run `yarn test:all` and `yarn lint`)

We appreciate your contribution!

Also, if you'd be interested in writing code like this for us more regularly, we're hiring:
https://careers.ghost.org

### CODE REVIEW

File: ghost/core/test/e2e-webhooks/pages.test.js [hunk 1]
```diff
@@ -120,6 +120,59 @@ describe('page.* events', function () {
...
+                        status: 'draft',
+                        slug: 'testing-page-edited-webhook'
+                    }
+                ]
+            })
+            .expectStatus(201);
+        
+        const id = res.body.pages[0].id;
+        const updatedPage = res.body.pages[0];
+        updatedPage.title = 'updated test page';
```
Comment (naz) [inline ghost/core/test/e2e-webhooks/pages.test.js:146]: Not something to change here necessarily. Just a small piece of feedback to improve future contributions. It's generally a more readable and maintainable solution when tests can achieve the needed state in as few steps as possible (unless it's some hacky shortcut that is vague to understand). In this case, just editing the title would have triggered the 'page.edited' event. Less code - better code 😊

**Review by naz — APPROVED** [review]
Thanks for contributing @shngt. Would love to see even more great work like this one 🙌

**Comment by shngt** [comment]
Thank you @naz! Also for the constructive feedback :)

**Comment by ErisDS** [comment]
Hi @shngt thank you so much for this PR. It has now been merged 🎉 and will appear in the next release of Ghost, which should be today.

I'm not sure if you found this through hacktoberfest, but I've added the `hacktoberfest-accepted` label to this PR to make sure it counts. We have plenty of open issues and ongoing projects for hacktoberfest and beyond. We would love to see you around again 👋

**Pull request merged**